resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/internal/properties/Bug468PerformanceTest.java

```diff
@@ -158,9 +159,18 @@ public void test() throws CoreException {
 		}, this.project, IWorkspace.AVOID_UPDATE, new NullProgressMonitor());
 
 		long maxTime = TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
+		long maxTimeMacOS = TimeUnit.MILLISECONDS.convert(75, TimeUnit.SECONDS);
```

Comment:
I would find it easier and more comprehensible to put these kinds of values into a `Duration`-typed variable. Currently, you have to scan all the code for assignments to `maxTime` and `maxTimeMacOS` to know what unit it uses. You can either fix this by putting the unit into the variable name or by using a type that handles the units internally, like `Duration` does. Then you can simply say `Duration maxTimeMacOs = Duration.ofSeconds(75)` and say `maxTimeMacOs.toMillis()` in case you need them in milliseconds.

Reply:
75s is not enough. A single test that wastes 90s is not good. please rewrite the test to take <1s. meanwhile it could be disabled.

Reply:
Agreed. Temporary disablement is better than temporary relaxation of the timeout value.
@jukzi Do you want to provide a PR to temporarily disable the test or shall I?

Reply:
please disable - or even remove the test.

Reply:
-> #891
Has to be replaced soon, so I think it's fine to keep it disabled and replace it rather than removing and re-adding it.